Rakefile

```diff
@@ -303,6 +327,7 @@ namespace :install do
   tasks << :blockly_symlink
   tasks << :apps if CDO.build_apps
   tasks << :shared if CDO.build_shared_js
+  tasks << :code_studio
```

Comment:
[Commit 550b9ce](https://github.com/code-dot-org/code-dot-org/commit/550b9ce6d0bf47dba9a75631d89ae9fc52cc6e0f) gates this task on configuration and set appropriate defaults for different environments.